<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>sign_form</class>
 <widget class="QMainWindow" name="sign_form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>485</width>
    <height>360</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>485</width>
    <height>360</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>485</width>
    <height>360</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>회원가입</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
	background-color: rgb(56, 58, 89);	
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>16</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>56</width>
      <height>16</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;아이디&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>71</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>71</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;패스워드&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>56</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;이메일&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>56</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;성별&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>56</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>56</width>
      <height>21</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>56</width>
      <height>21</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">	font: 75 9pt &quot;CookieRun Bold&quot;;
	color:white;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:10pt; font-weight:600;&quot;&gt;닉네임&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ID_textBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>20</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="PW_textBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Nick_Name_textBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="Email_textBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>170</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Nick_Name">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>120</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(135, 135, 181);
	border-radius:10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
}

QPushButton:hover {
	background-color: rgb(69, 57, 136);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>중복확인</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Double_Check">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>20</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(135, 135, 181);
	border-radius:10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
}

QPushButton:hover {
	background-color: rgb(69, 57, 136);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>중복확인</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Email_comboBox">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>170</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{

	background-color: rgb(135, 135, 181);
	border-radius:10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
}

QComboBox:hover {
	background-color: rgb(69, 57, 136);
}
QComboBox:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>@naver.com</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>@gmail.co.kr</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>@daum.net</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="Sign_In">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>270</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{

	background-color: rgb(135, 135, 181);
	border-radius:10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
}

QPushButton:hover {
	background-color: rgb(69, 57, 136);
}
QPushButton:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <property name="text">
     <string>가입하기</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Gender_comboBox">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>220</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{

	background-color: rgb(135, 135, 181);
	border-radius:10px;
	font: 75 9pt &quot;CookieRun Bold&quot;;
}

QComboBox:hover {
	background-color: rgb(69, 57, 136);
}
QComboBox:pressed {	
	background-color: rgb(85, 170, 255);
}</string>
    </property>
    <item>
     <property name="text">
      <string>남</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>여</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>485</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
